apps/E2E/src/RadioGroupV1/specs/RadioGroupV1.spec.win.ts

```diff
@@ -71,56 +100,80 @@ describe('RadioGroupV1 Functional Testing', async () => {
     await RadioGroupV1Page.resetRadioGroupSelection();
   });
 
-  it('Click on a Radio and ensure it changes state from unselected -> selected', async () => {
+  it('Click on a Radio. Validate that it changes state from unselected to selected.', async () => {
     /* Validate the Radio is not initially selected */
-    await expect(await RadioGroupV1Page.isRadioSelected(RadioSelector.Second)).toBeFalsy();
+    await expect(await RadioGroupV1Page.isRadioSelected(Radio.Second)).toBeFalsy(
+      'Expected radio #2 to unselected at test start, but #2 was initially selected.',
+    );
 
     /* Click on the Radio to select it */
-    await RadioGroupV1Page.clickRadio(RadioSelector.Second);
-    await RadioGroupV1Page.waitForRadioSelected(RadioSelector.Second, PAGE_TIMEOUT);
+    await RadioGroupV1Page.click(RadioGroupV1Page.getRadio(Radio.Second));
 
     /* Validate the Radio is selected */
-    await expect(await RadioGroupV1Page.isRadioSelected(RadioSelector.Second)).toBeTruthy();
+    await expect(
+      await RadioGroupV1Page.waitForRadioSelected(Radio.Second, 'Expected radio #2 to be selected by click, but #2 remained unselected.'),
+    ).toBeTruthy();
+
     await expect(await RadioGroupV1Page.didAssertPopup()).toBeFalsy(RadioGroupV1Page.ERRORMESSAGE_ASSERT);
   });
 
-  it('Keyboard to Radio and check for Selection state', async () => {
+  it('Press forward "Arrow Key" on a selected Radio. Validate adjacent Radio state changes from unselected to selected.', async () => {
     // Presses the ArrowDown key while the first (A) Radio is selected
-    await RadioGroupV1Page.sendKey(Keys.ARROW_DOWN, RadioSelector.First);
-    await RadioGroupV1Page.waitForRadioSelected(RadioSelector.Second, 5000);
+    await RadioGroupV1Page.sendKeys(RadioGroupV1Page.getRadio(Radio.First), [Keys.ARROW_DOWN]);
 
     /* Validate the Radio is selected */
-    await expect(await RadioGroupV1Page.isRadioSelected(RadioSelector.Second)).toBeTruthy();
+    await expect(
+      await RadioGroupV1Page.waitForRadioSelected(
+        Radio.Second,
+        'Expected radio #2 to be selected by a "DOWN ARROW" input from radio #1, but #2 remained unselected.',
```

Comment:
"Navigated to Radio #2 via 'down' arrow key - should be selected"

We don't have to add the "but #2 remained unselected".

The error message should describe what we **expect** to happen. Then, if someone sees this failure, they'll see that this expectation failed.

We don't want to provide a double-negative - that could be confusing